shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitVoltageMapResult.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.shortcircuit;
+
+/**
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class ShortCircuitVoltageMapResult {
```

Comment:
I think that we don't need this class. In my mind, a selective short circuit calculation will provide a list, a map (or a monitoring as we have in security analysis) of `ShortCircuitBusResult`. In each bus result, we will find voltage magnitudes and angles by phase, and the values for the symmetrical calculations. This bus result will be increased with phase to phase results too. This has to be specified with @JB-H.